constraint/pkg/client/client.go

```diff
@@ -358,12 +363,15 @@ func (c *Client) AddTemplate(ctx context.Context, templ *templates.ConstraintTem
 		CRD:      artifacts.crd,
 		Targets:  []string{artifacts.targetHandler.GetName()},
 	}
+	if _, ok := c.constraints[artifacts.gk]; !ok {
```

Comment:
Silly question - what happens if multiple ConstraintTemplates are defined with the same constraint GroupKind? Is that prevented anywhere?
Do we need to include the ConstraintTemplate name in the key for this map?

Reply:
Constraint Templates must be named after the constraint kind, which forces them to be singletons.